pages/home_page.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, driver):
 
 
     def create_new_report(self, name, details):
-        # wait = WebDriverWait(self.driver, 10)
-        # element = wait.until(EC.visibility_of_element_located(By.CSS_SELECTOR, 'div.MuiBox-root.css-1ykox53 > button:nth-child(2) > svg'))
         open_create_item_modal = self.driver.find_element(By.CSS_SELECTOR, 'div.MuiBox-root.css-1ykox53 > button:nth-child(2) > svg').click()
```

Comment:
you cannot hard code this, the selector generated by mui is not guaranteed to be "css-1ykox53" every time the dom elements render.

Reply:
can you add an ID for that button, please?